kiosk/src/Components/ScanQR.tsx

```diff
@@ -15,14 +16,33 @@ const ScanQR: React.FC<IProps> = ({ kiosk }) => {
     const screenWidth = window.screen.width;
     const phoneWidth = screenWidth < 500;
     const kioskId = urlHashList?.[1];
-    const [scannerVisible, setScanner] = useState(false);
+    const [scannerVisible, setScannerVisible] = useState(false);
     const [linkError, setLinkError] = useState(false);
+    const [linkVisible, setLinkVisible] = useState(false);
+    const qrReaderRendered = useRef(null);
+    const [html5QrCode, setQrCode] = useState<undefined | Html5Qrcode>();
```

Comment:
I know it's long, but should you follow convention here and have it be `setHtml5QrCode`?